sub.go

```diff
@@ -22,6 +22,7 @@ type SubParams struct {
 	BillingCycleAnchor          int64             `form:"billing_cycle_anchor"`
 	BillingCycleAnchorNow       bool              `form:"-"` // See custom AppendTo
 	BillingCycleAnchorUnchanged bool              `form:"-"` // See custom AppendTo
+	CancelAtPeriodEnd           bool              `form:"cancel_at_period_end"`
```

Comment:
the issue here is that this only works for the value `true`. It does not work for false as it will be ignored. You have to do an ugly hack like `NoProrate` in that same struct.